<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerConnectionDialog</class>
 <widget class="QDialog" name="ServerConnectionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Server Connection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>225</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="sbPort">
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="edPassword">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="edMountpoint"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Mountpoint:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edName"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="edAddress">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="cbType">
        <item>
         <property name="text">
          <string>Icecast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shoutcast</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="edUser"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Encoder</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>124</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>64</number>
        </property>
        <property name="maximum">
         <number>320</number>
        </property>
        <property name="singleStep">
         <number>32</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Samplerate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QComboBox" name="cbEncoderSampleRate">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>22050</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>44100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>96000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QSlider" name="slEncoderBitRate">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>64</number>
        </property>
        <property name="maximum">
         <number>320</number>
        </property>
        <property name="singleStep">
         <number>32</number>
        </property>
        <property name="pageStep">
         <number>32</number>
        </property>
        <property name="sliderPosition">
         <number>128</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="cbEncoder">
        <item>
         <property name="text">
          <string>Lame MP3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ogg Vorbis</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Encoder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Channels</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="cbEncoderChannels">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of channels used for streaming. If audio device is in mono mode, streaming switches to mono too. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edName</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>edAddress</tabstop>
  <tabstop>sbPort</tabstop>
  <tabstop>edPassword</tabstop>
  <tabstop>edMountpoint</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbEncoder</tabstop>
  <tabstop>slEncoderBitRate</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>cbEncoderSampleRate</tabstop>
  <tabstop>cbEncoderChannels</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServerConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServerConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slEncoderBitRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slEncoderBitRate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
